python_modules/dagster/dagster/_daemon/asset_daemon.py

```diff
@@ -384,6 +389,14 @@ def core_loop(
                         thread_name_prefix="asset_daemon_worker",
                     )
                 )
+                num_submit_workers = self._settings.get("num_submit_workers")
```

Comment:
there will need to be a corresponding internal PR for this i imagine?

Reply:
https://github.com/dagster-io/internal/pull/12675/files yep!